jib-core/src/main/java/com/google/cloud/tools/jib/frontend/BuildStepsRunner.java

```diff
@@ -166,6 +156,13 @@ private static void handleRegistryUnauthorizedException(
     }
   }
 
+  private static String capitalizeFirstLetter(String string) {
```

Comment:
Could use `StringUtils#capitalize()` instead?

Reply:
I'm trying to avoid having to add the Apache Commons library just for this function, and preferring Guava over Apache Commons.